### PR TITLE
Improved login page (droped registration page)

### PR DESCRIPTION
**Pull request by dufajlusi**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ################### DataSource Configuration ##########################
 
 jdbc.driverClassName=com.mysql.jdbc.Driver
-jdbc.url=jdbc:mysql://10.0.5.165:3306/teamodoro?characterEncoding=UTF-8
+jdbc.url=jdbc:mysql://localhost:3306/teamodoro?characterEncoding=UTF-8
```
Comment (PavleP) [inline src/main/resources/application.properties]: Current address is 10.0.5.185

File: src/main/resources/messages_en.properties [hunk 2]
```diff
@@ -1,13 +1,14 @@
+login.login=LOGIN
+login.register=REGISTER
 login.registration=Teamodoro registration
 login.sign.out=Sign out
-login.firstname=First (Given) Name
-login.lastname=Last (Family) Name
+login.firstname=First Name
+login.lastname=Last Name
```
Comment (PavleP) [inline src/main/resources/messages_en.properties]: It is not 100% clear what does it mean First and Last name and this the reason why I have put Given and Family in the brackets. I propose to keep this specifiers in the labels.

**Comment by stanislavgrujic** [comment]
Please resolve merge conflicts.

File: src/main/resources/application.properties (hunk 1 above)
Comment (dufajlusi) [inline src/main/resources/application.properties]: ok tnx

File: src/main/resources/messages_en.properties (hunk 2 above)
Comment (dufajlusi) [inline src/main/resources/messages_en.properties]: Most the web pages on login or register pages have "Last Name" and "First name"... like Facebook, Google...

**Pull request merged**